doma-spring-boot-samples/doma-spring-boot-sample-testcontainers/pom.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>doma-spring-boot-sample-testcontainers</artifactId>
+    <packaging>jar</packaging>
+
+    <name>doma-spring-boot-sample-testcontainers</name>
+    <description>Demo project for Spring Boot with Doma and TestContainers</description>
+
+    <parent>
+        <groupId>org.seasar.doma.boot</groupId>
+        <artifactId>doma-spring-boot-samples</artifactId>
+        <version>2.5.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.seasar.doma</groupId>
+            <artifactId>doma-core</artifactId>
+            <version>${doma.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.seasar.doma</groupId>
+            <artifactId>doma-processor</artifactId>
+            <version>${doma.version}</version>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>org.seasar.doma.boot</groupId>
+            <artifactId>doma-spring-boot-starter</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
+        <!-- PostgreSQL Driver -->
+        <dependency>
+            <groupId>org.postgresql</groupId>
+            <artifactId>postgresql</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+        <!-- H2 for local development -->
```

Comment:
Thanks to Testcontainers support. PostgreSQL can be used for local development as well.
H2 is no longer needed in this project.